examples/example-gateway/config/production.yaml

```diff
@@ -90,3 +90,5 @@ grpc.clientServiceNameMapping:
   echo: echo
 router.whitelistedPaths:
   - /path/whitelisted
+service.shadow.request.header: x-shadow-request
```

Comment:
I think these configs should go in the gateway config rather than env based config, see another similar header config example here: https://github.com/uber/zanzibar/blob/master/examples/example-gateway/build.yaml#L20

Reply:
Hi @rpatali , Actually I took the reference of **http.clients.requestUUIDHeaderKey: x-request-uuid**  and "**"sidecarRouter.default.http.callerHeader": "RPC-Caller"**" the way it implemented here [](https://github.com/uber/zanzibar/blob/master/examples/example-gateway/config/production.yaml#L53), but yes your point is valid that this is not environment specific and it should be present while gateway is bootstraping.
I tried to put the shadow header key in the build.yaml of example gateway but I unable to fetch it through static config.

Reply:
Request changes have been incorporated.